src/Orleans.Clustering.Consul/Orleans.Clustering.Consul.csproj

```diff
@@ -7,7 +7,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <AssemblyName>Orleans.Membership.Consul</AssemblyName>
+    <AssemblyName>Orleans.Clustering.Consul</AssemblyName>
     <RootNamespace>OrleansConsulUtils</RootNamespace>
```

Comment:
why is rootnamespace for this assembly Orleans.ConsulUtils, instead of Orleans.ConsulClustering?

Reply:
Because we decided to not to modify namespaces in existing assemblies.